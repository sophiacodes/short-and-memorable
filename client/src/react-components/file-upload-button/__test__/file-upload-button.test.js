import React from 'react';
import renderer from 'react-test-renderer';
import FileUploadButton from '../';
 
it('<FileUploadButton> component to render correctly when disabled', () => {
    const props = {
        className: "file-upload-button",
        id: "file-upload",
        disabled: true,
        onChange: jest.fn()
    };
    const tree = renderer
        .create(
            <FileUploadButton {...props} />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
 
it('<FileUploadButton> component to render correctly when undisabled', () => {
    const props = {
        className: "file-upload-button",
        id: "file-upload",
        disabled: false,
        onChange: jest.fn()
    };
    const tree = renderer
        .create(
            <FileUploadButton {...props} />
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});