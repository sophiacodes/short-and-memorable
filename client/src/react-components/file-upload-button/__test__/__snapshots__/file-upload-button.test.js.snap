// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FileUploadButton> component to render correctly when disabled 1`] = `
Array [
  <input
    className="file-upload-button"
    disabled={true}
    id="file-upload"
    onChange={[Function]}
    type="file"
  />,
  <label
    htmlFor="file-upload"
  >
    Choose file
  </label>,
]
`;

exports[`<FileUploadButton> component to render correctly when undisabled 1`] = `
Array [
  <input
    className="file-upload-button"
    disabled={false}
    id="file-upload"
    onChange={[Function]}
    type="file"
  />,
  <label
    htmlFor="file-upload"
  >
    Choose file
  </label>,
]
`;
